src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -67,9 +67,9 @@ abstract class AbstractFunctionalTest extends Specification {
     }
 
     private void setupDockerServerUrl() {
-        dockerServerUrl = TestConfiguration.dockerServerUrl
+        dockerServerUrl = TestConfiguration.dockerHost
 
-        if(dockerServerUrl) {
+        if (dockerServerUrl) {
```

Comment:
Thanks for fixing the spacing issues ;) At some point use of the checkstyle plugin would benefit greatly.